components/salsa-macros/src/db.rs

```diff
@@ -135,10 +135,11 @@ impl DbMacro {
         };
 
         input.items.push(parse_quote! {
+            #[cold]
+            #[inline(never)]
```

Comment:
This was previously marked as `inline(always)`, but it's only called in the slow path for both single and multi-database use cases.